api/TODO.md

```diff
@@ -0,0 +1,11 @@
+# TODO
```

Comment:
Hmm, should we create a tracking issue for visibility? :thinking:

Reply:
This was just the quick and dirty approach so you can see which specific todo items are handled by each PR